### PR TITLE
#551/Use score component in Game component

### PR DESCRIPTION
**Pull request by fizcamposancos**
First part of #551

### CODE REVIEW

**Review by gbudau — APPROVED** [review]
Thank you for this feature! LGTM :tada:

**Comment by Damrod** [comment]
I like it for phone device, but I still think we need a different version for desktop. Also, I think you may have some conflicts with the reconciliation branch now. Not sure, though.

File: webapp/src/shared/components/Game/Game.tsx [hunk 1]
```diff
@@ -53,18 +57,22 @@ const Play = ({ gameId }: GameProps) => {
 
   return (
     <div className="game-multiplayer">
-      <h1 className="game-multiplayer-score heading-bold">
-        {isPlayerOne ? opponentScore : score}
-      </h1>
+      <div className="game-multiplayer-score">
+        <Score
+          playerOneAvatar={{ url: `${AVATAR_EP_URL}/${playerOne!.avatarId}` }}
```
Comment (Damrod) [inline webapp/src/shared/components/Game/Game.tsx:62]: can we add avatarX and avatarY, since we have them? Both for playerOne and playerTwo. Thank you!

Comment (Damrod) [reply webapp/src/shared/components/Game/Game.tsx:62]:
```diff
diff --git a/webapp/src/shared/components/Game/Game.tsx b/webapp/src/shared/components/Game/Game.tsx
index 847e958..e054235 100644
--- a/webapp/src/shared/components/Game/Game.tsx
+++ b/webapp/src/shared/components/Game/Game.tsx
@@ -59,10 +59,18 @@ const Play = ({ gameId }: GameProps) => {
     <div className="game-multiplayer">
       <div className="game-multiplayer-score">
         <Score
-          playerOneAvatar={{ url: `${AVATAR_EP_URL}/${playerOne!.avatarId}` }}
+          playerOneAvatar={{
+            url: `${AVATAR_EP_URL}/${playerOne!.avatarId}`,
+            XCoordinate: playerOne!.avatarX,
+            YCoordinate: playerOne!.avatarY,
+          }}
           playerOneScore={score}
           playerOneUserName={playerOne!.username}
-          playerTwoAvatar={{ url: `${AVATAR_EP_URL}/${playerTwo!.avatarId}` }}
+          playerTwoAvatar={{
+            url: `${AVATAR_EP_URL}/${playerTwo!.avatarId}`,
+            XCoordinate: playerTwo!.avatarX,
+            YCoordinate: playerTwo!.avatarY,
+          }}
           playerTwoScore={opponentScore}
           playerTwoUserName={playerTwo!.username}
         />
```

**Review by Damrod — APPROVED** [review]
besides the two comments, LGTM!

File: webapp/src/shared/components/Game/Game.tsx (hunk 1 above)
Comment (fizcamposancos) [reply webapp/src/shared/components/Game/Game.tsx]: You're right, thanks. Done

**Comment by fizcamposancos** [comment]
> I like it for phone device, but I still think we need a different version for desktop. Also, I think you may have some conflicts with the reconciliation branch now. Not sure, though.

I think so, I was trying some approaches yesterday but I am not convinced yet. I will think a little more about it

**Comment by fizcamposancos** [comment]
@vdedios What do you think? Is this PR interfering with the reconciliation you are working on? Should I merge it?

**Comment by vdedios** [comment]
> @vdedios What do you think? Is this PR interfering with the reconciliation you are working on? Should I merge it?

merge it without fear!

**Review by vdedios — APPROVED** [review]

**Pull request merged**